src/components/breadcrumb-bar/breadcrumb-bar.module.css

```diff
@@ -18,6 +18,8 @@
 }
 
 .listItem {
+  display: flex;
```

Comment:
🚧 / 🙋🏻 -- Was this added only to get the text to truncate?

Reply:
Yep! This was done cause `breadcrumbText` would otherwise render as `display: inline` (since it's a span), which means the `max-width` doesn't seem to work 👍

Reply:
Ok got it! Thank you for the extra background, I thought that's what was going on. I think rather than setting `display: flex` on the `.listItem` class, we should set `display: block` on the `.breadcrumbText` class. This will isolate changes to the single class that needs them, and I think it'll be clearer at a glance what CSS is needed for the desired effect.

Reply:
Well shoot, that'll mess up the `::after` elements and then `display: flex` would be needed anyways on `.listItem`. Nevermind! 😎

Reply:
I'll add a lil docs on this, your second comment sums it up great! 💖

Reply:
Nice! Greatly appreciated 🙌🏻